### DISCUSSION TITLE
How can I get payee and payer from order that has been paid, please?

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore.Commerce#445] [Q&A] [answered]
### Describe the bug

How can I get payee and payer from order that has been paid,please?

**Comment by sarahelsaig** [comment]
[`OrderPart.BillingAddress`](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/12135b37946a810df8f37c6cb1aa9922d5cec6ff/src/Libraries/OrchardCore.Commerce.Abstractions/Models/OrderPart.cs#L44C25-L44C39) contains the payer's name and address, while `OrderPart.Email`, `OrderPart.Phone` and `OrderPart.VatNumber` properties contain some other information about the payer.

OCC doesn't store any payee information. That's something the payment processor knows.

**Comment by effapp** [comment]
Hi @sarahelsaig ,
There is a charges section that includes payment infomation in OrderPart , I thought it should include payee and payer infomation.

**Reply by sarahelsaig** [reply, marked as answer]
No, charges are [`IPayment`](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/12135b37946a810df8f37c6cb1aa9922d5cec6ff/src/Libraries/OrchardCore.Commerce.Abstractions/Abstractions/IPayment.cs#L9)s. They only contain the kind of short information like in a credit card statement's line items.
